rs/execution_environment/benches/system_api/execute_update.rs

```diff
@@ -878,6 +878,7 @@ pub fn execute_update_bench(c: &mut Criterion) {
             Module::Test.from_ic0("mint_cycles", Param1(1_i64), Result::I64, Wasm64::Enabled),
             18000006,
         ),
+        // TODO?
```

Comment:
Does it suffice to add a block here in order to run benchmarks? How do I run them? Should I commit the results?